cmd/earthly/root_cmds.go

```diff
@@ -173,6 +173,16 @@ func (app *earthlyApp) rootCmds() []*cli.Command {
 					Usage:       "Reset cache entirely by wiping cache dir",
 					Destination: &app.pruneReset,
 				},
+				&cli.StringFlag{
+					Name:        "target-size",
+					Usage:       "Prune cache to specified size, starting from oldest",
+					Destination: &app.pruneTargetSize,
+				},
+				&cli.DurationFlag{
+					Name:        "keep-duration",
```

Comment:
this seems easier to type:
```suggestion
					Name:        "age",
```